osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -156,7 +156,7 @@ private void load(AudioManager audio, IdleTracker idleTracker, GameHost host)
 
         private void onMultiplayer()
         {
-            if (!api.IsLoggedIn)
+            if (!api.IsLoggedIn || api.State.Value != APIState.Online)
```

Comment:
Seems a little weird to check both `IsLoggedIn` and state - at this point just the latter might be enough? Is this future-proofing and at some point we'll want to allow being online (as in there being a physical connection to the internet) but not logged in? Because right now I'm pretty sure you can't have the latter without the former.

Reply:
probably can remove the first.

Reply:
As a follow-up thought, the notification message should probably be reworded to "You gotta be online..." to match, to avoid user confusion (they're possibly logged in already after all, it's just their connection that's dying). I'll grab both things.